From 8ef68aadaa3aa29bc2661ab44db4ddc50e77cef5 Mon Sep 17 00:00:00 2001
From: Tomi Valkeinen <tomi.valkeinen@ideasonboard.com>
Date: Wed, 4 Oct 2023 11:25:16 +0300
Subject: [PATCH] media: rp1: cfe: Fix verbose debug print

Switch a debug print from cfe_dbg() to cfe_dbg_verbose() as it will be
printed often while streaming.

Signed-off-by: Tomi Valkeinen <tomi.valkeinen@ideasonboard.com>
---
 drivers/media/platform/raspberrypi/rp1_cfe/cfe.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/media/platform/raspberrypi/rp1_cfe/cfe.c
+++ b/drivers/media/platform/raspberrypi/rp1_cfe/cfe.c
@@ -975,8 +975,8 @@ static void cfe_buffer_queue(struct vb2_
 
 	if (!cfe->job_queued && cfe->job_ready &&
 	    test_all_nodes(cfe, NODE_ENABLED, NODE_STREAMING)) {
-		cfe_dbg("Preparing job immediately for channel %u\n",
-			node->id);
+		cfe_dbg_verbose("Preparing job immediately for channel %u\n",
+				node->id);
 		cfe_prepare_next_job(cfe);
 	}
 
